src/views/SystemRepositoryTable/index.tsx

```diff
@@ -126,56 +122,93 @@ export const SystemRepositoryTable = () => {
     return classnames(marginClassName, 'fa fa-caret-up');
   };
 
+  const filterBookmarks = (
```

Comment:
This could probably use a comment indicating that it's temporary; I'm assuming that when backend support is implemented, the filtering will be done server-side. (@jstrang-oddball, can you confirm?)

Reply:
Jeremy implied that the schema for the bookarks will only contain the id, which needs to be matched up with the payload from CEDAR.  That's what the filtering is for.  I wasn't under the impression that the database would store all the info in the bookmark